### PR TITLE
Remove imports of global module

### PR DESCRIPTION
**Pull request by ibgreen**
@abmai @jdeerhake

### CODE REVIEW

File: src/overlays/draggable-points-overlay.js [hunk 1]
```diff
@@ -25,7 +25,8 @@ import config from '../config';
 import ViewportMercator from 'viewport-mercator-project';
 
 import Immutable from 'immutable';
-import document from 'global/document';
+import {document} from '../utils/globals'
+    ;
```
Comment (abmai) [inline src/overlays/draggable-points-overlay.js]: spacing fix here (should be on previous line)

File: src/overlays/scatterplot-overlay.js [hunk 2]
```diff
@@ -17,12 +17,12 @@
...
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
-
 import {PropTypes, Component, createElement} from 'react';
 import ViewportMercator from 'viewport-mercator-project';
 
 import Immutable from 'immutable';
-import window from 'global/window';
+import {window} from '../utils/globals'
+    ;
```
Comment (abmai) [inline src/overlays/scatterplot-overlay.js]: spacing fix here

File: examples/choropleth-overlay/choropleth-overlay.js [hunk 3]
```diff
@@ -19,12 +19,14 @@
...
 import React, {PropTypes, Component} from 'react';
 import ViewportMercator from 'viewport-mercator-project';
-import window from 'global/window';
 import {extent} from 'd3-array';
 import {scaleLinear} from 'd3-scale';
 import {geoPath, geoTransform} from 'd3-geo';
 import Immutable from 'immutable';
 
+import {window} from '../utils/globals'
+    ;
```
Comment (abmai) [inline examples/choropleth-overlay/choropleth-overlay.js]: spacing

File: deprecated/map-interactions.js [hunk 4]
```diff
@@ -22,8 +22,8 @@
 // https://github.com/mapbox/mapbox-gl-js/blob/master/js/ui/handler/scroll_zoom.js
 import {PropTypes, PureComponent, createElement} from 'react';
 import autobind from '../utils/autobind';
-import document from 'global/document';
-import window from 'global/window';
+import {document, window} from '../utils/globals'
+    ;
```
Comment (abmai) [inline deprecated/map-interactions.js]: spacing

**Comment by ibgreen** [comment]
Sigh, the wonders of search-and-replace. 

Fixed it, but a little worried that the linter didn't catch it. Wonder if we have a bad setup - or did this just happen to represent legal code per our rules?

**Comment by jdeerhake** [comment]
Thanks!

**Pull request merged**